attributes/default.rb

```diff
@@ -93,7 +93,7 @@
 
 # platform family specific attributes
 case node['platform_family']
-when 'suse'
+when 'suse', 'opensuseleap'
```

Comment:
The platform_family for opensuseleap is actually suse, which includes SLES and openSUSE